### DISCUSSION TITLE
Upgrading EJBCA 7.4.3.2 Community: Backup, Upgrade Procedure, and Security Measures

### DISCUSSION THREAD

**Discussion by ramiabdallah1512** [Keyfactor/ejbca-ce#516] [Q&A]
I would like to update and upgrade EJBCA 7.4.3.2 Community to a more recent version. Could you please assist me in this step and provide your advice and recommendations for upgrading to the most stable version? Is it possible for you to help me and show me the backup procedure and the upgrade procedure to another more recent community version, and the steps to follow to perform this upgrade with full security?
thank you

**Comment by primetomas** [comment]
You should find all you need in the documentation.
https://doc.primekey.com/ejbca/ejbca-installation/upgrading-ejbca
